deployment/helm/eclipse-ditto/files/openapi/ditto-api-2.yml

```diff
@@ -0,0 +1,4134 @@
+# Copyright (c) 2017-2018 Bosch Software Innovations GmbH.
```

Comment:
I think we shouldn't duplicate the swagger documentation. How about a symbolic link?

Reply:
Thanks for the hint. Prepare an additional commit on this PR.